cos-fleet-catalog-connectors/aws/aws-sns-0.1/pom.xml

```diff
@@ -36,7 +36,7 @@
                             <description>${project.description} sink</description>
                             <adapter>
                                 <prefix>aws</prefix>
-                                <name>${cos.connector.type}-sink</name>
+                                <name>Send data to an Amazon Simple Notification Service (SNS) topic.</name>
```

Comment:
This was changed in a different place. Should be in <description> a few lines above.

Reply:
Sorry about that - thank you for catching it